src/Bedrock.Framework.Experimental/Infrastructure/Endianness.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Infrastructure
+{
+    public static class Endianness
```

Comment:
Internal. Do you need this though?

Reply:
If I don't have this I end up with a lot of (SomeEnum) BinaryPrimatives.Reverse((ushort) someEnum);

Etc it's really ugly and this using generics goes down all the way to basically the bswap in all cases. Also in later code for Tls there are other structs (including int24) that are just unsupported otherwise